examples/accounts/scripts/run.sh

```diff
@@ -2,22 +2,30 @@
 
 set -e
 
+JSON="Content-Type: application/json"
+
+
+
 echo ""; echo "Step 1"; echo ""
-pact -a examples/accounts/scripts/01-system.yaml | curl -d @- http://localhost:8080/api/v1/send
-curl -d '{"requestKeys":["221b1ea8687ca43da7eacdedfffe5a590215c334482b1148d5166e9847ead25e4d3a9c711c85c0b5ab9b739f151b562f37e1e21f2c8bc1004a5765575069cd91"]}' -X POST http://localhost:8080/api/v1/poll
+pact -a examples/accounts/scripts/01-system.yaml | curl -H "$JSON" -d @- http://localhost:8080/api/v1/send
+sleep 1
+curl -H "$JSON" -d '{"requestKeys":["020184904af377b619b066a60b06e703f44cb908eb19de69008cfd188ee65725b8c7022818955d90e320e530deba5c6ae7094749efacabeca9037d88016588ce"]}' -X POST http://localhost:8080/api/v1/poll
```

Comment:
This returns `Failed reading: satisfy`. Is this expected?

Reply:
Not happening for me: <code>Step 1

{"requestKeys":["020184904af377b619b066a60b06e703f44cb908eb19de69008cfd188ee65725b8c7022818955d90e320e530deba5c6ae7094749efacabeca9037d88016588ce"]}
{"020184904af377b619b066a60b06e703f44cb908eb19de69008cfd188ee65725b8c7022818955d90e320e530deba5c6ae7094749efacabeca9037d88016588ce":{"result":{"status":"success","data":{"t":"t","v":"2017-10-31T12:00:00.000Z"}},"metaData":null,"txId":0}}
Step 2

{"requestKeys":["fa6491227ea73ac3cbe00e4b95b9628217ace7a8911e1165caf37440936869121678841eb3d2c2741ba71e4222357b7a853944245a59dcd3bdd4a52a49757c53"]}
{"fa6491227ea73ac3cbe00e4b95b9628217ace7a8911e1165caf37440936869121678841eb3d2c2741ba71e4222357b7a853944245a59dcd3bdd4a52a49757c53":{"result":{"status":"success","data":"TableCreated"},"metaData":null,"txId":1}}
Step 3

{"requestKeys":["b31faa1e98ec1b9a2c489f0c5ea019fcf985aab27c46bb778c3bff5fb0e92da3cb67f54004bcd7cb95831c89d4487c9fd84f7f6fa84c64d2e3916afae7dafa3e"]}
{"b31faa1e98ec1b9a2c489f0c5ea019fcf985aab27c46bb778c3bff5fb0e92da3cb67f54004bcd7cb95831c89d4487c9fd84f7f6fa84c64d2e3916afae7dafa3e":{"result":{"status":"success","data":"[{\"amount\": 100.0,\"balance\": 100.0,\"ccy\": \"USD\",\"data\": {\"note\": \"Admin account funding\"},\"date\": \"2017-10-31T12:00:00Z\",\"rowguard\": KeySet {keys: [7d0c9ba189927df85c8c54f8b5c8acd76c1d27e923abbf25a957afdf25550804],pred: keys-all}} {\"amount\": 0.0,\"balance\": 0.0,\"ccy\": \"USD\",\"data\": {\"note\": \"Created account\"},\"date\": \"2017-10-31T12:00:00Z\",\"rowguard\": KeySet {keys: [ac69d9856821f11b8e6ca5cdd84a98ec3086493fd6407e74ea9038407ec9eba9],pred: keys-all}}]"},"metaData":null,"txId":2}}
Step 4

{"requestKeys":["ec36c9d3dc51fe138b29d9a420b60b82a09bfbb129dd4d8fe2d154799b3a3694fd120aa02bed068f460e61954e8ba1dff7640d439b615c492bf312e2ab8ad5ec"]}
{"ec36c9d3dc51fe138b29d9a420b60b82a09bfbb129dd4d8fe2d154799b3a3694fd120aa02bed068f460e61954e8ba1dff7640d439b615c492bf312e2ab8ad5ec":{"result":{"status":"success","data":{"t":"o","v":{"balance":{"t":"d","v":{"m":875,"p":1}},"ccy":"USD"}}},"metaData":null,"txId":3}}
Step 5

{"status":"success","data":true}</code>

Reply:
(note using < code > blocks to wrap stuff thus the funny output above)

Reply:
This got resolved when I rebuilt pact with your branch